platform/security/core/security-core-impl/src/main/java/ddf/security/liberty/paos/impl/ResponseImpl.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package ddf.security.liberty.paos.impl;
+
+import java.util.List;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import org.opensaml.core.xml.XMLObject;
+import org.opensaml.core.xml.schema.XSBooleanValue;
+import org.opensaml.saml.common.AbstractSAMLObject;
+
+import ddf.security.liberty.paos.Response;
+
+public class ResponseImpl extends AbstractSAMLObject implements Response {
+
+    private String refToMessageID;
+
+    private String soap11Actor;
+
+    private XSBooleanValue soap11MustUnderstand;
+
+    protected ResponseImpl(@Nullable String namespaceURI, @Nonnull String elementLocalName,
+            @Nullable String namespacePrefix) {
+        super(namespaceURI, elementLocalName, namespacePrefix);
+    }
+
+    @Override
+    public String getRefToMessageID() {
+        return refToMessageID;
+    }
+
+    @Override
+    public void setRefToMessageID(String newRefToMessageID) {
+        refToMessageID = prepareForAssignment(refToMessageID, newRefToMessageID);
+    }
+
+    @Nullable
```

Comment:
I'll probably just leave them since they should never change and there will never be any other classes, just these two.